src/ReactGoogleCharts.tsx

```diff
@@ -50,6 +51,7 @@ export class Chart extends React.Component<
           chartVersion={chartVersion}
           mapsApiKey={mapsApiKey}
           onLoad={this.onLoad}
+          onLoadCallback={onLoad}
```

Comment:
Why not just call it inside `this.onLoad` ?

```js
            onLoad = (google: GoogleViz) => {
              if (this.props.onLoad) {
                this.props.onLoad(google);
              }

               if (this.isFullyLoaded(google)) {
                 this.onSuccess(google);
               } else {
            ...
```

Reply:
Hi, @dangreen , good catch